### PR TITLE
MIGRATIONS [DA-2523] - Cron Interval / W3SR Manifest

### PR DESCRIPTION
**Pull request by dtharpeuno**
## Resolves *[ticket DA-2523]*
https://precisionmedicineinitiative.atlassian.net/browse/DA-2523

## Description of changes/additions
For the GHR3 manifest generations, there are criteria for certain manifests (4) to run on a bi-weekly basis. This includes the logic for the first one. 

## Tests
- [x] unit tests

### CODE REVIEW

File: rdr_service/cron_default.yaml [hunk 1]
```diff
@@ -157,7 +157,7 @@ cron:
 - description: Genomic CVL W3SR Workflow
```
Comment (j-kanuch) [inline rdr_service/cron_default.yaml]: Can you throw something like (scheduled weekly, runs bi-weekly) in the description?

File: rdr_service/dao/genomics_dao.py [hunk 2]
```diff
@@ -1131,14 +1131,17 @@ def get_id(self, obj):
...
+            return session.query(
+                functions.max(GenomicJobRun.startTime)
+            ).filter(GenomicJobRun.jobId == job_id,
+                     GenomicJobRun.runStatus == GenomicSubProcessStatus.COMPLETED,
+                     and_(
+                         or_(
+                             GenomicJobRun.runResult == GenomicSubProcessResult.SUCCESS,
+                             GenomicJobRun.runResult == GenomicSubProcessResult.NO_FILES
+                         )
+                     )
```
Comment (j-kanuch) [inline rdr_service/dao/genomics_dao.py]: Could this be simplified to the following?:
`GenomicJobRun.runStatus == GenomicSubProcessStatus.COMPLETED,
GenomicJobRun.runResult.in_(GenomicSubProcessResult.SUCCESS, GenomicSubProcessResult.NO_FILES)`
In MySql, IN is faster than OR statements.

Comment (dtharpeuno) [reply rdr_service/dao/genomics_dao.py]: https://github.com/all-of-us/raw-data-repository/pull/2897/commits/18961eb43d857fcffaba5238da33f6afb4a37e09

File: rdr_service/cron_default.yaml (hunk 1 above)
Comment (dtharpeuno) [reply rdr_service/cron_default.yaml]: Sure

**Review by j-kanuch — APPROVED** [review]

**Review by kenny-skaggs — APPROVED** [review]

**Pull request merged**